### PR TITLE
feat: shipped latency tracking code in scripts

### PR DESCRIPTION
**Pull request by maateen**
This PR depends on [this PR](https://github.com/jina-ai/latency-tracking/pull/8).

# Changes

- feat: shipped latency tracking code in scripts
- re-enabled latency tracking
- added document, flow, documentarraymem benchmark
- fixed IndexError and ValueError for changes in Jina Core 2.0.6

# Sample Output

```json
[
  {
    "version": "2.0.6",
    "import_time": 0.296265542,
    "dam_extend_time": 7.218721834,
    "dam_extend_qps": 138528.67903705957,
    "avg_flow_time": 0.9953305937500012,
    "index_time": 16.240183541,
    "query_time": 127.89024562499999,
    "index_qps": 3694.5395258941426,
    "query_qps": 32.02746214132936
  }
]
```

# Sample Comment

## Latency summary
 Current PR yields:
  - 😶 **index QPS** at `3690`, delta to last 2.0.5 avg.: `+0%`
  - 😶 **query QPS** at `31`, delta to last 2.0.5 avg.: `-3%`
  - 😶 **dam extend QPS** at `124741`, delta to last 2.0.5 avg.: `-9%`
  - 😶 **avg flow time** within `0.9226` seconds, delta to last 2.0.5 avg.: `-7%`
  - 🐎🐎🐎🐎 `import jina` within **0.3489** seconds, delta to last 2.0.5 avg.: `+17%`

## Breakdown
|                        Version                         | Index QPS | Query QPS | DAM Extend QPS | Avg Flow Time (s) | Import Time (s) |
|--------------------------------------------------------|-----------|-----------|----------------|-------------------|-----------------|
|                        current                         |    3690   |     31    |     124741     |       0.9226      |      0.3489     |
| [`2.0.5`](https://github.com/jina-ai/jina/tree/v2.0.5) |    3694   |     32    |     138528     |       0.9953      |      0.2963     |

Backed by [latency-tracking](https://github.com/jina-ai/latency-tracking). Further commits will update this comment.

### CODE REVIEW

**Comment by deepankarm** [comment]
`scripts/latency-tracking/original/index-original` file is >25MB & `scripts/latency-tracking/original/query-original` is >4MB. This cannot be merged in core, as it increased the repo size significantly. Can we keep them in `latency-tracking` & clone from there during the run?

**Comment by maateen** [comment]
@deepankarm In that case, it would be hard for the developers to run and verify the latency tracking codes. Please allow me to figure out a workaround.

File: scripts/latency-tracking/app.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+        A dict mapping of dam extend time in seconds as float number.
+    """
+    dlist = []
+    dam_size = 1000000
+    dam = DocumentArrayMemmap(os.path.join(os.getcwd(), 'MyMemMap'))
+
+    for i in range(dam_size):
+        dlist.append(Document(text=f'This is the document number: {i}', ))
+
+    log.info("Benchmarking DAM extend")
```
Comment (JoanFM) [inline scripts/latency-tracking/app.py:90]: Please let's use `single quotes`

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: scripts/latency-tracking/app.py (hunk 1 above)
Comment (maateen) [reply scripts/latency-tracking/app.py:95]: Without double quotes, the script won't pass `black` check.

**Review by deepankarm — APPROVED** [review]

**Comment by deepankarm** [comment]
@nan-wang This needs your approval

**Review by JoanFM — APPROVED** [review]

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**